src/seismometer/controls/selection.py

```diff
@@ -181,7 +207,7 @@ def __init__(
         value: Optional[tuple[str | tuple]] = None,
```

Comment:
I think a comma instead of a "|" is appropriate

Reply:
Yes, FINC